### PR TITLE
HP 47 Contact Us Snippet to Homepage

### PR DESCRIPTION
**Pull request by Audiosutras**
https://caktus.atlassian.net/browse/HP-47

### CODE REVIEW

File: apps/hip/templates/hip/home_page.html [hunk 1]
```diff
@@ -21,8 +21,11 @@
       {% endfor %}
     </section>
 
-    <section class="columns">
-      <div class="column"></div>
+    <section class="columns is-vcentered is-desktop">
+      <div class="column is-6-desktop">
+        <h3 class="is-size-4 pl-4"><strong>Report A Disease</h3>
```
Comment (vkurup) [inline apps/hip/templates/hip/home_page.html:26]: the `<strong>` tag needs to be closed.

**Review by vkurup — APPROVED** [review]
:shipit:

File: apps/hip/templates/hip/home_page.html (hunk 1 above)
Comment (Audiosutras) [reply apps/hip/templates/hip/home_page.html:26]: thanks

Comment (Audiosutras) [reply apps/hip/templates/hip/home_page.html]: done

**Pull request merged**